docs/asciidoc/modules/ROOT/pages/virtual-resource/index.adoc

```diff
@@ -118,15 +120,15 @@ The apoc.dv.catalog.list procedure returns a list with all the existing Virtuali
 
```

Comment:
Cambiare 

"It takes no parameters."

con: 

It accepts one parameter: i.e. the database name where we want to use the resource (default is `'neo4j'`)